gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionSha256Sum=cb87f222c5585bd46838ad4db78463a5c5f3d336e5e2b98dc7c0c586527351c2
```

Comment:
Why?

Reply:
Woops, sorry will remove that in next push just using a push to move between PC's